applications/ShapeOptimizationApplication/python_scripts/algorithm_bead_optimization.py

```diff
@@ -177,6 +174,9 @@ def RunOptimizationLoop(self):
         is_max_total_iterations_reached = False
         previos_L = None
 
+        # Initialize ALPHA_MAPPED according to initial ALPHA values
+        self.mapper.Map(ALPHA, ALPHA_MAPPED)
```

Comment:
It's better to put this to the function InitializeOptimizationLoop, I guess. As your comment already suggests, this is part of the initialization.